app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt

```diff
@@ -247,10 +247,20 @@ fun MessagePreview.uiLastMessageContent(): UILastMessageContent {
                 is WithUser.TeamMemberRemoved -> UILastMessageContent.None // TODO
                 is WithUser.Text -> UILastMessageContent.SenderWithMessage(
                     sender = userUIText,
-                    message = UIText.DynamicString((content as WithUser.Text).messageBody),
+                    message = (content as WithUser.Text).messageBody.let { UIText.DynamicString(it) },
                     separator = ": "
                 )
 
+                is WithUser.Composite -> {
+                    val text = (content as WithUser.Composite).messageBody?.let { UIText.DynamicString(it) }
```

Comment:
isn;t this crash prone when content is not WithUser.Composite and we try to cast it ?

Reply:
it is impossible to crash this because of the `when` this code will run iff the content is Composite